packages/flutter/lib/src/material/about.dart

```diff
@@ -384,6 +463,122 @@ class AboutDialog extends StatelessWidget {
   }
 }
 
+class _AdaptiveAboutDialog extends AboutDialog {
+  const _AdaptiveAboutDialog({
+    super.key,
+    super.applicationName,
+    super.applicationVersion,
+    super.applicationIcon,
+    super.applicationLegalese,
+    super.children,
+  });
+
+  List<Widget>? _actions(BuildContext context) {
+    final ThemeData themeData = Theme.of(context);
+    final MaterialLocalizations localizations =
+        MaterialLocalizations.of(context);
```

Comment:
Prefer LHS and RHS in variable assignments on the same line as per the style guide.